### DISCUSSION TITLE
Custom landscape will not display horizon line (polygonal_horizon) in 21.2

### DISCUSSION THREAD

**Discussion by fuadramsey** [Stellarium/stellarium#1955] [General]
My custom landscape works fine in version 20.0, but in 21.2 it will not display the horizon line generated from the polygonal_horizon_list. 

It does display the custom landscape image. I tried this on several computers and Windows 10 and Linux Mint.

**Comment by gzotti** [comment]
If you have not done so, try to add a value

polygonal_angle_rotatez = 0.00001

See User Guide, sec. 7.1.2. 

Please report if this helps.

**Comment by fuadramsey** [comment]
That fixed it!

Thank you!
